src/python/pants/pantsd/pants_daemon.py

```diff
@@ -162,7 +163,7 @@ def _initialize_metadata(self) -> None:
         self.write_pid()
         self.write_process_name()
         self.write_fingerprint(ensure_text(self.options_fingerprint))
-        self._logger.debug(f"pantsd running with PID: {self.pid}")
+        self._logger.info(f"pantsd {VERSION} running with PID: {self.pid}")
```

Comment:
This occurs outside of any client context, so it only goes to `.pants.d/pants.log`.